CONTRIBUTING.md

```diff
@@ -59,6 +59,22 @@ During the collector release process, all `./.chloggen/*.yaml` files are transcr
 
 Alternately, copy `./.chloggen/TEMPLATE.yaml`, or just create your file from scratch.
 
+## Platform Agnostic Code
+
+In order to ensure compatibility with different operating systems, code should be platform-agnostic. Below are some guidelines to follow when writing platform-agnostic code:
+
+* Avoid using platform-specific libraries, features etc. Please opt for cross-platform solutions. 
+
+* Avoid hard-coding platform specific values. Use environment variables or configuration files for storing platform specific values.
```

Comment:
What sort of platform-specific values are you expecting here?  Examples of what to do and what not to do would be helpful.

Reply:
Added an example, let me know if that is helpful.